scripts/us_census/pep/monthly_population_estimate/download.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" A Script to download USA Census PEP monthly population data
+    from the URLS in provided json file.
```

Comment:
Beyond downloading it also does some transformations and produces xlsx file?

Also note this in README?

Reply:
Description has been updated